fe/fe-core/src/main/java/com/starrocks/catalog/HMSTable.java

```diff
@@ -0,0 +1,29 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.catalog;
+
+import com.starrocks.common.DdlException;
+import com.starrocks.external.hive.HiveColumnStats;
+import com.starrocks.external.hive.HivePartition;
+import com.starrocks.external.hive.HivePartitionStats;
+import com.starrocks.external.hive.HiveTableStats;
+
+import java.util.List;
+import java.util.Map;
+
+public interface HMSTable {
```

Comment:
HMSTable -> HiveMetaStoreTable,since someone may not know about hms represent hive metastore

Reply:
done